components/progressbar/stories/meter.template.js

```diff
@@ -18,9 +19,13 @@ export const Template = ({
 		typeof fill !== "undefined" ? `is-${fill}` : null,
 	].filter(Boolean),
 	size,
+	customStyles: {
+		...customStyles,
+	},
```

Comment:
```suggestion
	customStyles,
```
The spread operator isn't needed here.

Reply:
https://github.com/adobe/spectrum-css/pull/2929/commits/f2407b194a2c62c66a1b06924e5a4b154b0cbc62